website/docs/d/group.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "azuread"
+page_title: "Azure Active Directory: azuread_group"
+sidebar_current: "docs-azuread-datasource-azuread-group"
+description: |-
+  Gets information about an Azure Active Directory group.
+
+---
+
+# Data Source: azuread_group
+
+Gets information about an Azure Active Directory group.
+
+-> **NOTE:** If you're authenticating using a Service Principal then it must have permissions to both `Read directory data` within the `Windows Azure Active Directory` API.
```

Comment:
remove " both " - it seems misplaced if there is only one permission needed

Reply:
Thanks. Fixed in the latest commit.